sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -213,17 +220,21 @@ export default {
           "Thank you, your application has been submitted.",
         );
       } catch (error) {
-        const errorLabel = "Unexpected error!";
+        let errorLabel = "Unexpected error!";
         let errorMsg = "An unexpected error happen.";
-        [
+        const isCustomError = [
           INVALID_STUDY_DATES,
           OFFERING_START_DATE_ERROR,
           PIR_OR_DATE_OVERLAP_ERROR,
-        ].forEach((customError) => {
-          if (error.includes(customError)) {
-            errorMsg = error.replace(customError, "").trim();
-          }
-        });
+          APPLICATION_DATE_OVERLAP_ERROR,
+        ].some(
+          (errorCode: string) => errorCode === error.response.data?.errorType,
+        );
+        if (isCustomError) {
+          errorLabel = "Application Overlap";
```

Comment:
mm, according to 231 logic, all errors mentioned in the list i.e 226, will enter here right? or am I missing anything

Reply:
and for `INVALID_STUDY_DATES,
          OFFERING_START_DATE_ERROR,
          PIR_OR_DATE_OVERLAP_ERROR,` errors also the label will be `errorLabel = "Application Overlap";`

Reply:
same in file `sources/packages/web/src/views/institution/locations/program-info-request/LocationEditProgramInfoRequest.vue`